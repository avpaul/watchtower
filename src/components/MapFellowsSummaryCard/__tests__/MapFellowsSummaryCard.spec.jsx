import React from 'react';
import { shallow } from 'enzyme';
import MapFellowsSummary from '../index';
import FellowsSumaryMock from '../../../__mocks__/fellowSummary.json';

describe('Test fellowsSummaryCard snapshots', () => {
  const props = {
    fellowsSummaryCardDetails: FellowsSumaryMock,
    handleClick: jest.fn()
  };
  const wrapper = shallow(<MapFellowsSummary {...props} />);

  it('fellowSummaryCard should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
