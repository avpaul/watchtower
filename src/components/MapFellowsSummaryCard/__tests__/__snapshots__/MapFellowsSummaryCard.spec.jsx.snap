// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test fellowsSummaryCard snapshots fellowSummaryCard should render properly 1`] = `
<div
  className="ops-dashboard__fellows-summary"
>
  <div
    className="row m-0"
  >
    <FellowsSummaryCard
      devPulseAverage="0.66"
      ending="13-Apr-2019"
      id="-LHs4sEcIAVnPeKHTBGd"
      key="0"
      level="D0B"
      lmsOutputs="16/16"
      name="Genesis Kohler"
      onClick={[MockFunction]}
      picture="https://lorempixel.com/100/100/people/?40049"
      product="Watch Tower"
      started="12-Jan-2019"
      status="Off-Track"
    />
    <FellowsSummaryCard
      devPulseAverage="0.79"
      ending="11-Dec-2018"
      id="-LD3PA3VgP4qvRRORX08"
      key="1"
      level="D0B"
      lmsOutputs="N/A"
      name="Burley Hirthe"
      onClick={[MockFunction]}
      picture="https://lorempixel.com/100/100/people/?58306"
      product="Watch Tower"
      started="17-Sep-2018"
      status="Off-Track"
    />
    <FellowsSummaryCard
      devPulseAverage="N/A"
      ending="11-Mar-2019"
      id="-LJK2fj_k_R7aYBAq_AR"
      key="2"
      level="D0B"
      lmsOutputs="16/16"
      name="Roberta Swift"
      onClick={[MockFunction]}
      picture="https://lorempixel.com/100/100/people/?33359"
      product="Watch Tower"
      started="15-Dec-2018"
      status="On-Track"
    />
    <FellowsSummaryCard
      devPulseAverage="1"
      ending="11-Mar-2019"
      id="-LJK1qECFdekucrwqqkh"
      key="3"
      level="D0B"
      lmsOutputs="16/16"
      name="Waldo Schuster"
      onClick={[MockFunction]}
      picture="https://lorempixel.com/100/100/people/?18151"
      product="Watch Tower"
      started="15-Dec-2018"
      status="Off-Track"
    />
    <FellowsSummaryCard
      devPulseAverage="2"
      ending="8-Apr-2019"
      id="-LLUW2BwQiZS0j9KF10j"
      key="4"
      level="D0B"
      lmsOutputs="16/16"
      name="Teresa Macejkovic"
      onClick={[MockFunction]}
      picture="https://lorempixel.com/100/100/people/?95393"
      product="Watch Tower"
      started="14-Jan-2019"
      status="On-Track"
    />
    <FellowsSummaryCard
      devPulseAverage="1.13"
      ending="16-Jan-2019"
      id="-LEnW5RRn0IkO8VjnMma"
      key="5"
      level="D0B"
      lmsOutputs="N/A"
      name="Maddison Kiehn"
      onClick={[MockFunction]}
      picture="https://lorempixel.com/100/100/people/?59737"
      product="Watch Tower"
      started="15-Oct-2018"
      status="Off-Track"
    />
  </div>
</div>
`;
